Extension to read and ingest Delta Lake tables (#15755)
apache/druid: docs/ingestion/input-sources.md

 to `true` to enable a compatibility mode where the timestampSpec is ignored.
:::

The [secondary partitioning method](native-batch.md#partitionsspec) determines the requisite number of concurrent worker tasks that run in parallel to complete ingestion with the Combining input source.
Set this value in `maxNumConcurrentSubTasks` in `tuningConfig` based on the secondary partitioning method:
- `range` or `single_dim` partitioning: greater than or equal to 1
- `hashed` or `dynamic` partitioning: greater than or equal to 2

> For more information on the `maxNumConcurrentSubTasks` field, see [Implementation considerations](native-batch.md#implementation-considerations).

Reviewer: seems unrelated to the PR

Author: Yeah, not directly related to this extension, but I noticed that this text was nested inside the Iceberg section: [Iceberg Filter Object](https://druid.apache.org/docs/latest/ingestion/input-sources#iceberg-filter-object) and seemed irrelevant. So I moved it to the Druid Input source, where it seems more relevant.

## Iceberg input source

:::info

> You need to include the [`druid-iceberg-extensions`](../development/extensions-contrib/iceberg.md) as an extension to use the Iceberg input source.

Reviewer 2: ```suggestion
To use the Iceberg input source, load the extension [`druid-iceberg-extensions`](../development/extensions-contrib/iceberg.md).
```

|type|Set this value to `not`.|yes|
|filter|The iceberg filter on which logical NOT is applied|yes|

## Delta Lake input source

:::info

> You need to include the [`druid-deltalake-extensions`](../development/extensions-contrib/delta-lake.md) as an extension to use the Delta Lake input source.

Reviewer 2: ```suggestion
To use the Delta Lake input source, load the extension [`druid-deltalake-extensions`](../development/extensions-contrib/delta-lake.md).
```
